app/src/androidTest/java/ch/epfl/sdp/blindly/match/TestMainToMatch.kt

```diff
@@ -0,0 +1,38 @@
+package ch.epfl.sdp.blindly.match
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.intent.Intents.*
+import androidx.test.espresso.intent.matcher.IntentMatchers.hasComponent
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import ch.epfl.sdp.blindly.R
+import ch.epfl.sdp.blindly.main_screen.MainScreen
+import dagger.hilt.android.testing.HiltAndroidRule
+import dagger.hilt.android.testing.HiltAndroidTest
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+@HiltAndroidTest
+class TestMainToMatch {
+
+    @get:Rule
+    val activityRule = ActivityScenarioRule(MainScreen::class.java)
+
+    @get:Rule
+    var hiltRule = HiltAndroidRule(this)
+
+    @Before
+    fun setup() {
+        hiltRule.inject()
+    }
```

Comment:
I don't think you need Hilt there. JUnit would suffice.

Reply:
If I remove it the test doesn't pass